<<<<<<< HEAD
CRITICAL - 2023-07-31 22:25:05 --> syntax error, unexpected token "<<"
in APPPATH\Controllers\Home.php on line 81.
 1 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\pi\\app\\Controllers\\Home.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Home')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Home')
 4 SYSTEMPATH\CodeIgniter.php(890): class_exists('\\App\\Controllers\\Home', true)
 5 SYSTEMPATH\CodeIgniter.php(484): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-07-31 22:25:45 --> syntax error, unexpected token "<<"
in APPPATH\Controllers\Home.php on line 81.
 1 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\pi\\app\\Controllers\\Home.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Home')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Home')
 4 SYSTEMPATH\CodeIgniter.php(890): class_exists('\\App\\Controllers\\Home', true)
 5 SYSTEMPATH\CodeIgniter.php(484): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-07-31 22:25:47 --> syntax error, unexpected token "<<"
in APPPATH\Controllers\Home.php on line 81.
 1 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\pi\\app\\Controllers\\Home.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Home')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Home')
 4 SYSTEMPATH\CodeIgniter.php(890): class_exists('\\App\\Controllers\\Home', true)
 5 SYSTEMPATH\CodeIgniter.php(484): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-07-31 22:25:49 --> syntax error, unexpected token "<<"
in APPPATH\Controllers\Home.php on line 81.
 1 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\pi\\app\\Controllers\\Home.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Home')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Home')
 4 SYSTEMPATH\CodeIgniter.php(890): class_exists('\\App\\Controllers\\Home', true)
 5 SYSTEMPATH\CodeIgniter.php(484): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-07-31 23:32:37 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\pi\system\Database\MySQLi\Connection.php:295
=======
ERROR - 2023-07-31 22:57:06 --> mysqli_sql_exception: Unknown column 'cnpjcpf' in 'field list' in C:\xampp\htdocs\pi\system\Database\MySQLi\Connection.php:295
>>>>>>> 62adeb405684a8b96ae4f337874ec7a33d0e13f0
Stack trace:
#0 C:\xampp\htdocs\pi\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `fo...', 0)
#1 C:\xampp\htdocs\pi\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fo...')
#2 C:\xampp\htdocs\pi\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fo...')
#3 C:\xampp\htdocs\pi\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fo...', Array, false)
#4 C:\xampp\htdocs\pi\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\pi\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\pi\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\pi\app\Controllers\Home.php(209): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\pi\system\CodeIgniter.php(932): App\Controllers\Home->salvarfornecedor()
#9 C:\xampp\htdocs\pi\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\pi\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\pi\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
<<<<<<< HEAD
CRITICAL - 2023-07-31 23:32:37 --> Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (\'48496\', \'tee\', \'7juiçu\', \'+5533988753282\', \'fabyolacampos20@gmail.com\', \'fem\', \'uh\', \'virgem\', \'MG\', \'39630000\', \'5\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (\'48496\', \'tee\', \'7juiçu\', \'+5533988753282\', \'fabyolacampos20@gmail.com\', \'fem\', \'uh\', \'virgem\', \'MG\', \'39630000\', \'5\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (\'48496\', \'tee\', \'7juiçu\', \'+5533988753282\', \'fabyolacampos20@gmail.com\', \'fem\', \'uh\', \'virgem\', \'MG\', \'39630000\', \'5\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (:cnpjcpf:, :nome_fornecedor:, :rua:, :telefone:, :email:, :sexo:, :bairro:, :cidade:, :uf:, :cep:, :numero:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Home.php(209): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->salvarfornecedor()
10 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
11 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-07-31 23:36:19 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\pi\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\pi\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `fo...', 0)
#1 C:\xampp\htdocs\pi\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fo...')
#2 C:\xampp\htdocs\pi\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fo...')
#3 C:\xampp\htdocs\pi\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fo...', Array, false)
#4 C:\xampp\htdocs\pi\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\pi\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\pi\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\pi\app\Controllers\Home.php(209): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\pi\system\CodeIgniter.php(932): App\Controllers\Home->salvarfornecedor()
#9 C:\xampp\htdocs\pi\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\pi\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\pi\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-07-31 23:36:19 --> Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (\'48496\', \'tee\', \'7juiçu\', \'+5533988753282\', \'fabyolacampos20@gmail.com\', \'fem\', \'uh\', \'virgem\', \'MG\', \'39630000\', \'5\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (\'48496\', \'tee\', \'7juiçu\', \'+5533988753282\', \'fabyolacampos20@gmail.com\', \'fem\', \'uh\', \'virgem\', \'MG\', \'39630000\', \'5\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (\'48496\', \'tee\', \'7juiçu\', \'+5533988753282\', \'fabyolacampos20@gmail.com\', \'fem\', \'uh\', \'virgem\', \'MG\', \'39630000\', \'5\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (:cnpjcpf:, :nome_fornecedor:, :rua:, :telefone:, :email:, :sexo:, :bairro:, :cidade:, :uf:, :cep:, :numero:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Home.php(209): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->salvarfornecedor()
10 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
11 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-07-31 23:37:40 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\pi\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\pi\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `fo...', 0)
#1 C:\xampp\htdocs\pi\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fo...')
#2 C:\xampp\htdocs\pi\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fo...')
#3 C:\xampp\htdocs\pi\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fo...', Array, false)
#4 C:\xampp\htdocs\pi\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\pi\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\pi\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\pi\app\Controllers\Home.php(209): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\pi\system\CodeIgniter.php(932): App\Controllers\Home->salvarfornecedor()
#9 C:\xampp\htdocs\pi\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\pi\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\pi\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-07-31 23:37:40 --> Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (\'48496\', \'tee\', \'7juiçu\', \'+5533988753282\', \'fabyolacampos20@gmail.com\', \'fem\', \'uh\', \'virgem\', \'MG\', \'39630000\', \'5\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (\'48496\', \'tee\', \'7juiçu\', \'+5533988753282\', \'fabyolacampos20@gmail.com\', \'fem\', \'uh\', \'virgem\', \'MG\', \'39630000\', \'5\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (\'48496\', \'tee\', \'7juiçu\', \'+5533988753282\', \'fabyolacampos20@gmail.com\', \'fem\', \'uh\', \'virgem\', \'MG\', \'39630000\', \'5\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (:cnpjcpf:, :nome_fornecedor:, :rua:, :telefone:, :email:, :sexo:, :bairro:, :cidade:, :uf:, :cep:, :numero:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Home.php(209): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->salvarfornecedor()
10 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
11 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-07-31 23:38:23 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\pi\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\pi\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `fo...', 0)
#1 C:\xampp\htdocs\pi\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fo...')
#2 C:\xampp\htdocs\pi\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fo...')
#3 C:\xampp\htdocs\pi\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fo...', Array, false)
#4 C:\xampp\htdocs\pi\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\pi\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\pi\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\pi\app\Controllers\Home.php(209): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\pi\system\CodeIgniter.php(932): App\Controllers\Home->salvarfornecedor()
#9 C:\xampp\htdocs\pi\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\pi\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\pi\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-07-31 23:38:23 --> Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (\'\', \'\', \'\', \'\', \'\', \'Masculino\', \'\', \'\', \'AC\', \'\', \'\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (\'\', \'\', \'\', \'\', \'\', \'Masculino\', \'\', \'\', \'AC\', \'\', \'\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (\'\', \'\', \'\', \'\', \'\', \'Masculino\', \'\', \'\', \'AC\', \'\', \'\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (:cnpjcpf:, :nome_fornecedor:, :rua:, :telefone:, :email:, :sexo:, :bairro:, :cidade:, :uf:, :cep:, :numero:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Home.php(209): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->salvarfornecedor()
10 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
11 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-07-31 23:40:12 --> Deletes are not allowed unless they contain a "where" or "like" clause.
in SYSTEMPATH\Database\BaseBuilder.php on line 2767.
 1 SYSTEMPATH\Model.php(456): CodeIgniter\Database\BaseBuilder->delete()
 2 SYSTEMPATH\BaseModel.php(1065): CodeIgniter\Model->doDelete('0', false)
 3 APPPATH\Controllers\Home.php(235): CodeIgniter\BaseModel->delete('0')
 4 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->excluirfornecedor()
 5 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-07-31 23:42:40 --> Error connecting to the database: mysqli_sql_exception: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente in C:\xampp\htdocs\pi\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\pi\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'bdfinae', 3306, '', 0)
#1 C:\xampp\htdocs\pi\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pi\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pi\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fo...', Array, false)
#4 C:\xampp\htdocs\pi\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\pi\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\pi\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\pi\app\Controllers\Home.php(209): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\pi\system\CodeIgniter.php(932): App\Controllers\Home->salvarfornecedor()
#9 C:\xampp\htdocs\pi\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\pi\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\pi\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente in C:\xampp\htdocs\pi\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\pi\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pi\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pi\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fo...', Array, false)
#3 C:\xampp\htdocs\pi\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\pi\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\pi\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\pi\app\Controllers\Home.php(209): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\pi\system\CodeIgniter.php(932): App\Controllers\Home->salvarfornecedor()
#8 C:\xampp\htdocs\pi\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp\htdocs\pi\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pi\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-07-31 23:42:40 --> Unable to connect to the database.
Main connection [MySQLi]: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (:cnpjcpf:, :nome_fornecedor:, :rua:, :telefone:, :email:, :sexo:, :bairro:, :cidade:, :uf:, :cep:, :numero:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Home.php(209): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->salvarfornecedor()
 8 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-07-31 23:43:07 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\pi\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\pi\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `fo...', 0)
#1 C:\xampp\htdocs\pi\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fo...')
#2 C:\xampp\htdocs\pi\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fo...')
#3 C:\xampp\htdocs\pi\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fo...', Array, false)
#4 C:\xampp\htdocs\pi\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\pi\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\pi\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\pi\app\Controllers\Home.php(209): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\pi\system\CodeIgniter.php(932): App\Controllers\Home->salvarfornecedor()
#9 C:\xampp\htdocs\pi\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\pi\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\pi\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-07-31 23:43:07 --> Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (\'12345\', \'faby\', \'Tomires\', \'+5533988753282\', \'fabyolacampos20@gmail.com\', \'Feminino\', \'bela\', \'virgem\', \'MG\', \'39630000\', \'12\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (\'12345\', \'faby\', \'Tomires\', \'+5533988753282\', \'fabyolacampos20@gmail.com\', \'Feminino\', \'bela\', \'virgem\', \'MG\', \'39630000\', \'12\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (\'12345\', \'faby\', \'Tomires\', \'+5533988753282\', \'fabyolacampos20@gmail.com\', \'Feminino\', \'bela\', \'virgem\', \'MG\', \'39630000\', \'12\')')
=======
CRITICAL - 2023-07-31 22:57:06 --> Unknown column 'cnpjcpf' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (\'704654\', \'alison\', \'Rua Medeiro SILVA\', \'3243423\', \'alison@gmaol.com\', \'Juridico/Não definido\', \'São Geraldo\', \'Araçuaí\', \'TO\', \'39600000\', \'342\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (\'704654\', \'alison\', \'Rua Medeiro SILVA\', \'3243423\', \'alison@gmaol.com\', \'Juridico/Não definido\', \'São Geraldo\', \'Araçuaí\', \'TO\', \'39600000\', \'342\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (\'704654\', \'alison\', \'Rua Medeiro SILVA\', \'3243423\', \'alison@gmaol.com\', \'Juridico/Não definido\', \'São Geraldo\', \'Araçuaí\', \'TO\', \'39600000\', \'342\')')
>>>>>>> 62adeb405684a8b96ae4f337874ec7a33d0e13f0
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fornecedor` (`cnpjcpf`, `nome_fornecedor`, `rua`, `telefone`, `email`, `sexo`, `bairro`, `cidade`, `uf`, `cep`, `numero`) VALUES (:cnpjcpf:, :nome_fornecedor:, :rua:, :telefone:, :email:, :sexo:, :bairro:, :cidade:, :uf:, :cep:, :numero:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Home.php(209): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->salvarfornecedor()
10 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
11 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
